.gitignore

```diff
@@ -8,3 +8,4 @@
 .externalNativeBuild
 .cxx
 local.properties
+app/schemas
```

Comment:
We can add stuff to the file `.gitignore` so that git ignores it!

Here I'm ignoring the schema file that Room generates. I felt like it was basically a build file so it felt weird to add it to version control.